tensorflow_datasets/core/dataset_builder.py

```diff
@@ -205,6 +205,8 @@ def __init__(self, data_dir=None, config=None, version=None):
     else:  # Use the code version (do not restore data)
       self.info.initialize_from_bucket()
 
+    self._code_dir = os.path.dirname(inspect.getfile(self.__class__))
```

Comment:
It seems that we want to access the file location without having to instantiate the class, so this should be a class member (e.g. set through https://docs.python.org/3/reference/datamodel.html#object.__init_subclass__, or through a memoized class property)